packages/vscode-extension/src/telemetry/extTelemetryEvents.ts

```diff
@@ -388,6 +388,14 @@ export enum TelemetryProperty {
   CopilotChatParticipantId = "copilot-chat-participant-id",
   CopilotChatLocation = "copilot-chat-location",
   CopilotChatCompleteType = "copilot-chat-complete-type",
+  CopilotMatchResultType = "copilot-match-result-type",
+  CopilotChatBlockReason = "copilot-chat-block-reason",
+  CopilotChatRelatedSampleName = "copilot-chat-related-sample-name",
+  CopilotChatCodeClassAndMembers = "copilot-chat-code-class-and-members",
+  CopilotChatTimeToFirstToken = "copilot-chat-time-to-first-token",
+  CopilotChatTotalTokensPerSecond = "copilot-chat-total-tokens-per-second",
+  CopilotChatResponseTokensPerRequest = "copilot-chat-response-tokens-per-request",
+  CopilotChatTotalTokens = "copilot-chat-total-tokens",
```

Comment:
There is existing CopilotChatTokenCount, so we should just use that.

Reply:
the existing name could not distinguish request token and response token. I have added 2 new telemetry name to make it clear